app/views/posts/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1> posts </h1>
+<ul>
+<% @posts.each do |post| %>
+<li>
+<h3><%= post.title %></h3>
+<p> <%= post.content %> </p>
+<% user = User.find(post.user_id)%>
+<% if @current_user %>
```

Comment:
Here it is clearer is you use a `signed_in?` method to check this condition.

Reply:
This is outdated, I now use the #logged_in? method.